wiki/Main_Page/de.md

```diff
@@ -0,0 +1,89 @@
+
+</div>
+<div class="wiki-main-page-panel">
+
+# Spielverlauf
+
+[Spielmodis](/wiki/Game_mode): [osu!](/wiki/Game_mode/osu!) • [osu!taiko](/wiki/Game_mode/osu!taiko) • [osu!catch](/wiki/Game_mode/osu!catch) • [osu!mania](/wiki/Game_mode/osu!mania)
+
+[Beatmap](/wiki/Beatmap) • [Hit-Objekte](/wiki/Hit_object) • [Mods](/wiki/Game_modifier) • [Punktzahl](/wiki/Score) • [Wiederholung](/wiki/Replay) • [Multi](/wiki/Multi)
+
+</div>
+<div class="wiki-main-page-panel">
+
+# [Beatmap editor](/wiki/Beatmap_Editor)
+
+Sections: [Zusammenstellung](/wiki/Beatmap_Editor/Compose) • [Design](/wiki/Beatmap_Editor/Design) • [Timing](/wiki/Beatmap_Editor/Timing) • [Song-Einstellung](/wiki/Beatmap_Editor/Song_Setup)
+
+Komponenten: [AiMod](/wiki/Beatmap_Editor/AiMod) • [Schlagschnellteiler](/wiki/Beatmap_Editor/Beat_Snap_Divisor) • [Schlagschnellteiler](/wiki/Beatmap_Editor/Distance_Snap) • [Menu](/wiki/Beatmap_Editor/Menu) • [SB Ladung](/wiki/Beatmap_Editor/SB_Load) • [Zeitleisten](/wiki/Beatmap_Editor/Timelines)
+
+[Beatmapping](/wiki/Beatmapping) • [Schwierigkeitsgrade](/wiki/Beatmap/Difficulty) • [Mapping Techniken](/wiki/Mapping_Techniques) • [Storyboarding](/wiki/Storyboarding)
+
+</div>
+<div class="wiki-main-page-panel">
+
+# Beatmap-Einreichung und Ranking
+
+[Einreichen](/wiki/Submission) • [Modding](/wiki/Modding) • [Ranking-Verfahren](/wiki/Beatmap_ranking_procedure) • [Mappers' Gilde](/wiki/Mappers_Guild) • [Project Loved](/wiki/Project_Loved)
+
+[Ranking-Kriterien](/wiki/Ranking_Criteria): [osu!](/wiki/Ranking_Criteria/osu!) • [osu!taiko](/wiki/Ranking_Criteria/osu!taiko) • [osu!catch](/wiki/Ranking_Criteria/osu!catch) • [osu!mania](/wiki/Ranking_Criteria/osu!mania)
+
+</div>
+<div class="wiki-main-page-panel">
+
+# Community
+
+[Turniere](/wiki/Tournaments) • [Skinning](/wiki/Skinning) • [Projekte](/wiki/Projects) • [Anleitungen](/wiki/Guides) • [osu!dev Discord server](/wiki/osu!dev_Discord_server) • [Wie Sie helfen können](/wiki/How_You_Can_Help!) • [Glossar](/wiki/Glossary)
+
+</div>
+<div class="wiki-main-page-panel">
+
+# Personen
+
+[Das Team](/wiki/People/The_Team): [Entwickler](/wiki/People/The_Team/Developers) • [Globales Moderationsteam](/wiki/People/The_Team/Global_Moderation_Team) • [Support Team](/wiki/People/The_Team/Support_Team) • [Nominierungsbewertungsteam](/wiki/People/The_Team/Nomination_Assessment_Team) • [Beatmap-Nominatoren](/wiki/People/The_Team/Beatmap_Nominators) • [osu! Alumni](/wiki/People/The_Team/osu!_Alumni) • [Projekt Loved Team](/wiki/People/The_Team/Project_Loved_Team)
+
+Organisationen: [osu! UCI](/wiki/Organisations/osu!_UCI)
+
+[Community Beiträger](/wiki/People/Community_Contributors) • [Benutzer mit besonderen Titeln](/wiki/People/Users_with_unique_titles)
```

Comment:
_Beiträger_ sounds strange, but might be correct. I've simply never heard this word before. I'd just go for _Beitragende_ or 
```suggestion
[Community-Mitwirkende](/wiki/People/Community_Contributors) • [Benutzer mit besonderen Titeln](/wiki/People/Users_with_unique_titles)
```